x/mongo/driver/topology/rtt_monitor_test.go

```diff
@@ -219,6 +222,106 @@ func TestRTTMonitor(t *testing.T) {
 			rtt.reset()
 		}
 	})
+
+	// GODRIVER-2464
+	// Test that the RTT monitor can continue monitoring server RTTs after an operation gets stuck.
+	// An operation can get stuck if the server or a proxy stops responding to requests on the RTT
+	// connection but does not close the TCP socket, effectively creating an operation that will
+	// never complete.
+	t.Run("stuck operations time out", func(t *testing.T) {
+		t.Parallel()
```

Comment:
Marking this test as parallel probably means no other test within this package can use `localhost:0`?

Reply:
Using address `localhost:0` actually causes the listener to pick an available port. From the [net.Listen](https://pkg.go.dev/net#Listen) documentation:
> If the port in the address parameter is empty or "0", as in "127.0.0.1:" or "[::1]:0", a port number is automatically chosen.

Many listeners can use address `localhost:0` at the same time because they will each get different unused port numbers.